doc/userguide/rules/http-keywords.rst

```diff
@@ -190,6 +190,12 @@ characters %20 in a uri. This means matching on the uri.raw. The
 uri.raw and the normalized uri are separate buffers. So, the uri.raw
 inspects the uri.raw buffer and can not inspect the normalized buffer.
 
+.. note:: uri.raw never has any spaces in it.
+          In case of a request line ``GET /uid=0(root) gid=0(root) HTTP/1.1``,
```

Comment:
Suggest surrounding  uri.raw, http.uri.raw and http.protocol with double back-ticks

Suggest slight tweak to wording: "With this request line ``GET ..."

Reply:
ok thanks